client/src/index.html

```diff
@@ -4,14 +4,24 @@
 		<meta charset="UTF-8" />
 		<meta name="viewport" content="initial-scale=1, width=device-width" />
 		<title>Starter Kit</title>
+		<title>Module Deadline Tracker</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
A HTML document should only have 1 title tag

Reply:
done